src/boundary_conditions/include/grins/pressure_pinning.h

```diff
@@ -56,6 +57,13 @@ namespace GRINS
 		     const std::string& physics_name );
     ~PressurePinning();
 
+    //! Check the mesh to ensure pin location is found
+    /*! We are assuming here that the pin location is on the boundary
+        so we only check the boundary. If the pin location is found, we
+        set _pin_location_found to true (for assertion later). If not found,
```

Comment:
Oh crap, thanks. I wrote that before the code, realized we wanted to do element checks and then forgot to update the comment.